src/app/meshes/views/MeshListView.vue

```diff
@@ -1,124 +1,123 @@
 <template>
-  <RouteView
-    v-slot="{ route }"
-    name="mesh-list-view"
+  <DataSource
+    v-slot="{ data: me }: MeSource"
+    src="/me"
   >
```

Comment:
**Question**: I’m not sure I follow why we need this. Why can’t we just use the `pageSize` from the route query parameters (and default to a constant)? Why does this need data-sourcing?

Reply:
Because we are going to persist this value to localStorage when the user selects it from the menu they don't have to keep doing it every time they come to the page.